src/app/core/auth/auth-request.service.ts

```diff
@@ -19,18 +19,17 @@ export class AuthRequestService {
 
   constructor(@Inject(GLOBAL_CONFIG) protected EnvConfig: GlobalConfig,
               protected halService: HALEndpointService,
-              protected responseCache: ResponseCacheService,
               protected requestService: RequestService) {
   }
 
   protected fetchRequest(request: RestRequest): Observable<any> {
-    return this.responseCache.get(request.href).pipe(
-      map((entry: ResponseCacheEntry) => entry.response),
+    return this.requestService.getByUUID(request.uuid).pipe(
+      getResponseFromEntry(),
       // TODO to review when https://github.com/DSpace/dspace-angular/issues/217 will be fixed
-      tap(() => this.responseCache.remove(request.href)),
+      // tap(() => this.responseCache.remove(request.href)),
```

Comment:
One last code question: Should we remove these two comments, since this PR claims to fix #217?  Is there any reason these two lines should be left around?